imu_driver:
  ros__parameters:
    i2c_address: 0x4b # 0x4a or 0x4b
    enable_imu: true
    enable_magnetometer: false
    imu_rate: 100
    magnetometer_rate: 100